lib/bin/repl.js

```diff
@@ -7,18 +7,22 @@
 // including this file, may be copied, modified, propagated, or distributed
 // except according to the terms contained in the LICENSE file.
 
-const should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
-require('../../test/assertions');
-
 const _ = require('lodash'); // eslint-disable-line import/no-extraneous-dependencies
 const { sql } = require('slonik');
 
 const container = require('../util/default-container');
 
 (async () => {
-  const context = { ..._.omit(container, 'with'), container, should, sql };
-  const contextKeys = Object.keys(context).sort();
-  console.log('Available vars:', contextKeys.join(', '));
+  const context = { ..._.omit(container, 'with'), container, sql };
+  const contextKeys = Object.keys(context);
+
+  try {
+    context.should = require('should'); // eslint-disable-line import/no-extraneous-dependencies
+    contextKeys.push('should'); // manually add due to should.js prototype pollution
```

Comment:
> prototype pollution causes us to specify `useGlobal: true`

`useGlobal: true` allows `should` to work as expected, because it changes the root `Object.prototype`:

```js
> Object.prototype.should
Assertion { obj: [Object: null prototype] {}, anyOne: false, negate: false, params: { actual: [Object: null prototype] {} } }
```

Reply:
After the discussion above, I see now how the prototype pollution affects the line that this new comment refers to.